src/plugins/transit-demand/TransitLayers.tsx

```diff
@@ -164,36 +165,41 @@ export default function Component({
   }, [links])
 
   // ----------------------------------------------
-  const slices = useMemo(() => {
-    // no boarding data? no pies.
-    if (!stopMarkers.length || !('boardings' in stopMarkers[0])) return []
-
-    const fullPies = stopMarkers.map(stop => {
-      let selectedLineStopBoardingsCount = 0
-      let selectedLineStopAlightingsCount = 0
-      Object.entries(transitLines).forEach(([key, line]) => {
-        const selectedPtLine = Object.values(stop.ptLines).find(
-          (ptLine) => (ptLine as PtLine).name === line.id
-        ) as PtLine | undefined;
-        
-        if (selectedPtLine) {
-          selectedLineStopBoardingsCount += selectedPtLine.b;
-          selectedLineStopAlightingsCount += selectedPtLine.a;
-        }
-      })
+  var slices = {}
```

Comment:
slices is an array, not an object. Change to `slices = []` (probably needs an "as any[]")